File: StaticDemo.java
package week_3;
class StaticDemo {
    
    static
    {
        System.out.println("first is the best.");
    }

    static int  x =5;
    static float  y =5;

    static int  meth(int a,int b)
    {
        int sum = a+b;
        return sum;
    }

    public static void main(String args[])
    {
        System.out.println(x + " " + y);
        System.out.println(meth(10,20));
    }
}

File: StaticDemo.cpp
#include<iostream>
using namespace std;
class StaticDemo
{
    static int a;
    public:
    static int meth(int a,int b)
    {
        cout << "First is the best."<< endl;
        return a+b;
    }

};
int StaticDemo::a = 10;
int main()
{
    cout << StaticDemo::meth(1,2) << endl;
    return 0;
}